src/bootstrap-table.js

```diff
@@ -734,7 +734,7 @@ class BootstrapTable {
 
     // Adding the button html to the final toolbar html when the showOption is true
     for (const button of opts.buttonsOrder) {
-      const showOption = opts[`show${button.charAt(0).toUpperCase()}${button.substring(1)}`]
+      const showOption = (opts[`show${button.charAt(0).toUpperCase()}${button.substring(1)}`] || opts[`show${button}`])
```

Comment:
should be `const showOption = opts[`show${button.charAt(0).toUpperCase()}${button.substring(1)}`]`

Reply:
I'm pretty sure we need that (same for the other comment)
Could you test your idea ?

If you think that is enough please tell it, then i test again why we need that.

Reply:
Example: https://live.bootstrap-table.com/code/wenzhixin/4840

Reply:
For what is this example ?
That example seems working, the buttons is created but its hidden because `show<buttonname>`  is `false`.

Reply:
Sorry, this example: https://live.bootstrap-table.com/code/wenzhixin/5067
`showBtnTest: false`